scripting/src/main/java/com/tersesystems/echopraxia/scripting/ScriptManager.java

```diff
@@ -70,7 +71,7 @@ public void setUserFunctions(Function<LoggingContext, List<ValueMapEntry>> userF
         findNumber("$.age") == 3 && findString("$.name") == "Will";
    }
   */
-  public boolean execute(boolean df, Level level, LoggingContext context) {
+  public boolean execute(boolean df, Level level, LoggingContextWithFindPathMethods context) {
```

Comment:
Need to make it explicit that scripting only works with the extended logging context.